datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -461,6 +467,67 @@ where
                     .map_err(|e| e.into_inner().unwrap_or(EthereumContractCallError::Timeout))
             })
     }
+
+    /// Request blocks by hash through JSON-RPC.
+    fn load_blocks_rpc(
```

Comment:
Do we need this as a dedicated method? Can't `load_blocks` be used without caring whether or not blocks come from the Ethereum node or the database?

Reply:
You mean we could inline this into `load_blocks`? Sure, but this code is very similar to `load_block_ptrs_rpc`, so I put them as their own methods and next to each other, as a reminder that if we figure out how to optimize one of them, we should do the same optimization to the other.

Reply:
Ok, I see it's an internal method anyway. I'll mark this point as resolved.